.circleci/config.yml

```diff
@@ -21,6 +21,14 @@ aliases:
         only: /.*/
       branches:
         only: master
+  - &only_master_tags_and_p4_1090
+    filters:
+      tags:
+        only: /.*/
+      branches:
+        only:
+        - master
+        - p4-1090-send-azure-events
```

Comment:
this seems a bit clunky: a comment might help explain why only  this branch and master should be deployed?

Reply:
Its because I wanted to test deployments.